config/locales/server.en.yml

```diff
@@ -1248,6 +1248,7 @@ en:
     default_email_private_messages: "Send an email when someone messages the user by default."
     default_email_direct: "Send an email when someone quotes/replies to/mentions or invites the user by default."
     default_email_mailing_list_mode: "Send an email for every new post by default."
+    disable_mailing_list_mode: "Disable sending an email for every new post for all users"
```

Comment:
I think this should be something along the lines of `Disallow users from enabling mailing list mode`